debugging/debugging.md

```diff
@@ -35,11 +35,17 @@
 - [Get Started with Debugging JavaScript with Chrome Dev Tools (tutorial)](https://developers.google.com/web/tools/chrome-devtools/javascript/)
 
 ### Lesson
-
-[Debugging (video walkthrough of slides)](https://drive.google.com/file/d/1VMyGDG5CnNMkjCyaNx1LAjxTMKxC0QQ4/view?usp=sharing)
-
-[Debugging (slides)](https://docs.google.com/presentation/d/1Ol1as_RuxBpXMd4VxCUyvFVkjpcqAxg6B-c0Sl9KDLc/edit?usp=sharing)
-
+- Video walkthrough of slides [Debugging (12 mins watch)](https://drive.google.com/file/d/1VMyGDG5CnNMkjCyaNx1LAjxTMKxC0QQ4/view?usp=sharing)
+- Read through lesson slides [Debugging](https://docs.google.com/presentation/d/1Ol1as_RuxBpXMd4VxCUyvFVkjpcqAxg6B-c0Sl9KDLc/edit?usp=sharing)
+
+##### General Tips for Debugging (with VS Code)
+- When VS Code shows you a red squiggly underline then it's trying to help you so I insist you pay attention to the message.
+- If the red squiggly underline is under something that's clearly correct, like `color: red;` in a CSS file, then first check if it really is correct.
+- Pattern matching — It is a mechanism for checking a value against a pattern. In debugging you use pattern matching(inbuilt in VS Code) to match the syntax you write with the correct syntax which helps you to figure out errors in your code. For example, suppose you want to change the color of a text in CSS and you write `colour: blue;` then VS Code uses its pattern matching mechanism to match it again correct syntax and give you an error "check property spelling" as the correct form is `color: blue;`.
+- When you make a change on a file, and then visually when you refresh your page you don't see any changes then I suggest you go through the following points:
+    - Check whether you save the file? If you forget then I suggest you add autosave in VS Code, it will help you save the latest changes even if you forget.([How to add autosave in VS Code](https://code.visualstudio.com/docs/editor/codebasics#_save-auto-save))
+    - Make a temporary, very obvious change on your HTML page such as write some text in the top of the `body` or a prominent heading then save the file and reload the browser. Check whether you can see the text? If you can't see the new text, maybe you have the wrong file open, or the file didn't save, or something else very strange is happening.
```

Comment:
Same thing with: 'Check whether you can see the text?'  Turn it into a question or a statement; right now it is in between.

Reply:
Also for 
Check whether the thing you expected to turn red actually turns red?